todo_app/templates/index.html

```diff
@@ -5,11 +5,12 @@
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
+    <p class="lead">Current Loged in user: {{user1.name}} And ClientID: {{user1.id}}</p>
   </div>
   <form action=".\new" method="POST" >
     <div>
       <label for="name">Title: </label>
-      <input type="text" name="title" id="title" required> <input type="submit" value="Add Item">
+      <input type="text" name="title" id="title" required> <input type="submit" value="Add Item" {{role}}>
```

Comment:
If we do change the writer role the HTML could look a bit odd, so we could separate this effect from the role name like:
```suggestion
      <input type="text" name="title" id="title" required> 
      <input type="submit" value="Add Item" 
      {% if role != "writer" %}
          disabled
      {% endif %}
      >
```

Reply:
Made relevent changes and duplicated down the checking or role to be Writer where i have it checking for Disabled